assets/overlays/openstack-cinder/base/controller_privileged_binding.yaml

```diff
@@ -0,0 +1,14 @@
+# Allow the controller to run privileged pods
```

Comment:
Is that still necessary after https://github.com/openshift/csi-operator/pull/275/commits/c1dc7c651aee697601797a86d0690fc4c0fc38bb ?

Reply:
Whoops, missed this. Removed.